compose/config.go

```diff
@@ -19,6 +19,9 @@ type Config struct {
 	// HashCost sets the cost of the password hashing cost. Defaults to 12.
 	HashCost int
 
+	// AccessOnly sets whether the introspection endpoint only validates access tokens.
+	AccessOnly bool
```

Comment:
I think the name of this variable is misleading, how about `DisableRefreshTokenValidation`?